docs/IAS-XSUAA-token-xchange.md

```diff
@@ -7,19 +7,9 @@ The user is logged into a multitenant zone-enabled application that performs aut
 The application calls a service in the context of the current user and zone, by sending the IAS ID token for the user. That means the IAS ID token consists of `user_uuid` and `zone_uuid` claim.
 The SCP Identity Kernel service uses XSUAA internally and thus needs to exchange the ID token from IAS to an access token from XSUAA.
 
-#### Under the hood
-![IAS -> XSUAA token xchange flow diagram](token-xchange.png)
-
-Token of incoming request is checked whether it is a Xsuaa token, if it is Xsuaa token then this token proceeds forward to the token validation step. In case it is valid token, successful response is sent back, otherwise an unauthorized response is sent. 
-In case token is not Xsuaa token, application checks if token exchange is enabled, if it is, then Xsuaa access token is requested from Xsuaa instance using the POST request with Header `X-zid`:`zone_uuid`, `grant_type` = `jwt-bearer` and `assertion` = IAS token from the request. Upon successful token exchange between IAS and XSUAA, token gets validated and depending on validation result authorized or unauthorized response is sent back.
-In situation when incoming request doesn't contain Xsuaa token and token exchange is disabled, unauthorized response is sent back. 
+![token-xchange-setup](./images/TokenExchangeSetup.png)
```

Comment:
Please add a header 
## Setup 
And explain it a little bit: 
- App Router >= 9.x version mit Link auf https://npmjs.com/package/@sap/approuter
- Approuter needs to be bound to IAS 
- App needs to be bound to XSUAA
- explain xsuaa cross consumption (trust between IAS and XSUAA)